rockspecs/luv-scm-0.rockspec

```diff
@@ -79,6 +79,9 @@ build = {
             'userenv';
             'ws2_32';
             'advapi32';
+            'Dbghelp';
+            "Ole32";
+            "Shell32";
```

Comment:
Why are these needed?

Reply:
to fix https://ci.appveyor.com/project/racker-buildbot/luv/builds/47096188/job/9j5t4vt905amaa2f#L1222